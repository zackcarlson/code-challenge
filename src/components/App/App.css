.app-root {
  padding: 5rem 0 0 0;
  background: #efefef;
  min-height: 100vh;
}
