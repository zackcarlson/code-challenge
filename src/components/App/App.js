import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import "./App.css";
import Routes from "../../routes";
import ApolloContext from "../../state/apollo";

const App = () => (
  <div className="app-root">
    <ApolloContext>
      <Router>
        <Routes />
      </Router>
    </ApolloContext>
  </div>
);

export default App;
