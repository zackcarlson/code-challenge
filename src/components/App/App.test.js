import React from "react";
import renderer from "react-test-renderer";
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import App from "./App";

configure({ adapter: new Adapter() });

test("snapshot renders", () => {
  const component = renderer.create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
