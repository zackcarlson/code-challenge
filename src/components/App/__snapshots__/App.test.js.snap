// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot renders 1`] = `
<div
  className="app-root"
>
  <div>
    <nav
      className="nav-bar"
    >
      <ul>
        <li>
          <a
            className="logo-link"
            href="/"
            onClick={[Function]}
          >
            <img
              alt="New Engen Logo"
              className="logo-icon"
              src="NewEngen-Logo.svg"
            />
          </a>
        </li>
        <li>
          <a
            className="cart-link"
            href="/cart"
            onClick={[Function]}
          >
            <img
              alt="Cart Icon"
              className="cart-icon"
              src="CartIcon.svg"
            />
            <span
              className="cart-amount"
            />
          </a>
        </li>
      </ul>
    </nav>
    <div
      className="home-page"
    >
      <ul
        className="swatch-list"
      />
      <button
        onClick={[Function]}
        type="button"
      >
        Load More
      </button>
    </div>
  </div>
</div>
`;
